data/publisher/src/main/java/tech/pegasys/teku/data/publisher/GeneralMetricData.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.data.publisher;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class GeneralMetricData extends BaseMetricData {
```

Comment:
we've got `BaseMetricData`, `GeneralMetricData`, `DefaultMetricData`...
It looks like we only use `GeneralMetricData`...

it looks like maybe `GeneralMetricData` is more like `ValidatorMetricData`; and `DefaultMetricData` might be `BeaconNodeMetricData` or `MinimalMetricData`?

Reply:
Thanks for the input. I changed the `DefaultMetricData` to `MinimalMetricData`.